test-framework-docker/executor/src/main/java/com/containersolutions/mesoshelloworld/executor/FrameworkTask.java

```diff
@@ -0,0 +1,71 @@
+package com.containersolutions.mesoshelloworld.executor;
+
+import com.sun.net.httpserver.HttpExchange;
+import com.sun.net.httpserver.HttpHandler;
+import com.sun.net.httpserver.HttpServer;
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Replace this usage of Sun classes by ones from the Java API. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://ci.infra.container-solutions.com/sonar/coding_rules#rule_key=squid%3AS1191)